codestyle.cfg

```diff
@@ -0,0 +1,231 @@
+# Copyright The Mbed TLS Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Configuration options for Mbed TLS code style
+
+# Line length options
+
+# Limit lines to 80 characters (or try to)
+code_width = 80
+
+# Allow splitting long for statements between the condition statements
+ls_for_split_full = true
+
+# Allow splitting function calls between arguments
+ls_func_split_full = true
+
+
+input_tab_size = 4
+
+# Spaces-only indentation
+indent_with_tabs = 0
+
+indent_columns = 4
+
+# Indent 'case' 1 level from 'switch'
+indent_switch_case = indent_columns
+
+# Line-up strings broken by '\'
+indent_align_string = true
+
+# (When false) Don't align continued assignment statements but use an indent instead
+#indent_align_assign = false
+
+# Indent function call parameters instead of aligning them on the next line
+#indent_func_call_param = true
+
+# Indent function definition parameters instead of aligning them on the next line
+#indent_func_def_param = true
+
+# Indent function definition parameters instead of aligning them on the next line
+#indent_func_proto_param = true
+
+
+# K&R-like options
+
+# Braces on the same line (Egyptian-style braces)
+nl_enum_brace = remove
+nl_union_brace = remove
+nl_struct_brace = remove
+nl_do_brace = remove
+nl_if_brace = remove
+nl_for_brace = remove
+#nl_else_brace = remove # <-- Ignore else for now to allow macro-guard magic
```

Comment:
Do you have an estimate for how bad things go if we turn this on, which I think we do?